Projekty/Projekt1/Gruszkowska_Klaudia_Jamrozy_Bartosz/KM2.ipynb

```diff
@@ -0,0 +1,3476 @@
+{
```

Comment:
<p>Warto aby jednym zdaniem wprowadzić czytelnika na czym kolejna analiza będzie polegała i jaki jest jej cel (generalnie więcej opisów bo notebook pełni funkcję mini-raportu).</p><p> </p><p>Dobry kierunek prac, w kontekście KM3 należałoby popracować m.in. nad dobieraniem parametrów modeli (strojenie będzie omawiane na kolejnych zajęciach). Warto dodać macierze pomyłek (ang. confusion matrix). W przypadku gdy kod powtarza się bardzo wiele razy należy zastosować funkcje.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>